modular_skyrat/modules/sec_haul/code/misc/vending.dm

```diff
@@ -62,22 +63,39 @@
 					/obj/item/storage/backpack/satchel/sec = 3,
 					/obj/item/storage/backpack/duffelbag/sec = 3,
 					/obj/item/clothing/under/rank/security/officer = 10,
+					/obj/item/clothing/under/rank/security/peacekeeper/tactical = 4,
 					/obj/item/clothing/shoes/jackboots/security = 10,
 					/obj/item/clothing/head/security_garrison = 10,
 					/obj/item/clothing/head/security_cap = 10,
-					/obj/item/clothing/gloves/color/black/security = 10,
-					/obj/item/storage/belt/security/webbing/peacekeeper = 2,
-					/obj/item/storage/backpack/security/peacekeeper = 5,
-					/obj/item/storage/backpack/satchel/sec/peacekeeper = 5,
 					/obj/item/clothing/head/beret/sec/peacekeeper = 5,
-					/obj/item/clothing/under/rank/security/peacekeeper = 5,
-					/obj/item/clothing/under/rank/security/peacekeeper/tactical = 2,
+					/obj/item/clothing/gloves/color/black/security = 10,
```

Comment:
Keep the tactical outfit and the backpack, since they are favoured options

Reply:
tactical outfit is just moved up in the vendor to be in the same spot as all the other Under items